### PR TITLE
Adding AuML OD batch scoring notebook

### PR DESCRIPTION
**Pull request by shubhamiit**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Adding AutoML for Images Object Detection batch scoring notebook.

fixes #

### CODE REVIEW

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items -batch-scoring/coco2jsonl.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import json
```
Comment (nvijayrania) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items -batch-scoring/coco2jsonl.py:1]: [Minor] There is an extra space in the name of the folder.

**Review by nvijayrania — COMMENTED** [review]
Could you please update the scoring scripts screenshot from an OD run. This one has model_settings from classification.

**Review by nvijayrania — COMMENTED** [review]
[Minor] Please check the failing smoke tests

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items -batch-scoring/coco2jsonl.py (hunk 1 above)
Comment (shubhamiit) [reply sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items -batch-scoring/coco2jsonl.py]: Done

**Comment by shubhamiit** [comment]
> Could you please update the scoring scripts screenshot from an OD run. This one has model_settings from classification.

Done

**Review by nvijayrania — APPROVED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/image-object-detection-batch-scoring-non-mlflow-model.ipynb [hunk 2]
```diff
@@ -0,0 +1,1173 @@
...
+    "from azure.ai.ml import automl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2. Configure workspace details and get a handle to the workspace\n",
+    "\n",
+    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
```
Comment (sharma-riti) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/image-object-detection-batch-scoring-non-mlflow-model.ipynb:66]:
```suggestion
    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
```

**Review by sharma-riti — APPROVED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/image-object-detection-batch-scoring-non-mlflow-model.ipynb [hunk 3]
```diff
@@ -0,0 +1,1173 @@
+{
+ "cells": [
```
Comment (sharma-riti) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items-batch-scoring/image-object-detection-batch-scoring-non-mlflow-model.ipynb:2]: are we keeping both object detection and classification batch scoring notebooks?

**Pull request merged**